### PR TITLE
db: Change default DB to mysql

### PR DESCRIPTION
**Pull request by punfil**

### CODE REVIEW

File: assets/styles/themes/dark.css [hunk 1]
```diff
@@ -163,15 +163,15 @@ header #search_term, .sidebar-wrapper .header-logo, .tabs,
...
+    background-color: rgb(0, 0, 0);
 }
 
 .croppie-container .cr-resizer, .croppie-container .cr-viewport {
     box-shadow: 0 0 2000px 2000px #7d7f82;
 }
 
 .sidebar-element.separate-item {
-    border-top: 1px solid #7d7f82;
+    border-top: 1px solid tgb(0, 56, 200);
```
Comment (bartoszLesniewski) [inline assets/styles/themes/dark.css:174]: typo?

File: docker-compose.dist.yml [hunk 2]
```diff
@@ -7,50 +7,31 @@ services:
...
+            - DB_DRIVER=pdo_mysql
             - DB_NAME=koillection
-            - DB_HOST=postgresql
-            - DB_PORT=5432
-            - DB_USER=postgres
+            - DB_HOST=mysql
+            - DB_PORT=3306
+            - DB_USER=root
             - DB_PASSWORD=password
             - DB_VERSION=15
```
Comment (bartoszLesniewski) [inline docker-compose.dist.yml:21]: Database version 15 is related to postgres. The latest version of MySQL is 8.0.

File: docker-compose.dist.yml [hunk 3]
```diff
@@ -7,50 +7,31 @@ services:
...
             - PHP_TZ=Europe/Paris
```
Comment (bartoszLesniewski) [inline docker-compose.dist.yml:22]: Please change to Europe/Warsaw.

File: docker-compose.dist.yml [hunk 4]
```diff
@@ -7,50 +7,31 @@ services:
         container_name: koillection
         build:
             dockerfile: Dockerfile.dev
-        restart: always
```
Comment (bartoszLesniewski) [inline docker-compose.dist.yml:9]: Are you sure it should be removed? It works with this line as well.

File: docker-compose.dist.yml [hunk 5]
```diff
@@ -7,50 +7,31 @@ services:
...
         image: mysql:latest
-        command: --default-authentication-plugin=mysql_native_password
+        container_name: mysql
         environment:
-            - MYSQL_ROOT_PASSWORD=password
             - MYSQL_DATABASE=koillection
-
-        ports:
-            - 3306:3306
+            - MYSQL_ROOT_PASSWORD=password
```
Comment (bartoszLesniewski) [inline docker-compose.dist.yml:35]: The order doesn't matter here ;)

File: docker-compose.dist.yml [hunk 6]
```diff
@@ -7,50 +7,31 @@ services:
...
-            - POSTGRES_HOST_AUTH_METHOD=trust
-        ports:
-            - 5432:5432
-        volumes:
-            - "./docker/volumes/postgresql:/var/lib/postgresql/data"
-
     mysql:
-        container_name: mysql
         image: mysql:latest
-        command: --default-authentication-plugin=mysql_native_password
```
Comment (bartoszLesniewski) [inline docker-compose.dist.yml:47]: I'm not sure if this should be removed. From what I've read, missing this line may sometimes cause problems with MySQL 8.0, so please consider reverting (I tested with this line and it works, too).

File: docker-compose.dist.yml (hunk 4 above)
Comment (punfil) [reply docker-compose.dist.yml:9]: Yeah, it has been removed in the upstream. We will need to revisit all these when we sync with upstream.
If container exits with rc != 0 then restart the container.

File: docker-compose.dist.yml (hunk 5 above)
Comment (punfil) [reply docker-compose.dist.yml:35]: Aligned to docker-compose.yml for production and dev in upstream. Will be changed anyways when fork sync happens.

File: docker-compose.dist.yml (hunk 6 above)
Comment (punfil) [reply docker-compose.dist.yml:47]: Aligned to upstream, it's not used there. No problems observed with this line removed so far.

File: docker-compose.dist.yml (hunk 2 above)
Comment (punfil) [reply docker-compose.dist.yml:21]: why does it work then ;)
Seems like a placeholder, will create a separate commit

**Comment by punfil** [comment]
PR needs to be moved as now we merge to `dev` branch instead.

**Pull request closed without merging**